src/js/videojs-dash.js

```diff
@@ -40,13 +41,9 @@ class Html5DashJS {
     let manifestSource = source.src;
     this.keySystemOptions_ = Html5DashJS.buildDashJSProtData(source.keySystemOptions);
 
-    // Save the context after the first initialization for subsequent instances
-    Html5DashJS.context_ = Html5DashJS.context_ || {};
+    player.dash.mediaPlayer = dashjs.MediaPlayer().create();
```

Comment:
`this.mediaPlayer_.reset()` is necessary when changing sources from dash to something else.